frontend/src/component/common/TimeAgo/TimeAgo.tsx

```diff
@@ -4,47 +4,54 @@ import { formatDistanceToNow, secondsToMilliseconds } from 'date-fns';
 type TimeAgoProps = {
     date: Date | number | string | null | undefined;
     fallback?: string;
-    live?: boolean | number;
+    refresh?: boolean;
+    timeElement?: boolean;
 };
 
-const getRefreshInterval = (input?: boolean | number) => {
-    if (input === undefined) return secondsToMilliseconds(12);
-    if (input === false) return false;
-    if (input === true) return secondsToMilliseconds(12);
-    return secondsToMilliseconds(input);
-};
-
-const TimeAgo: FC<TimeAgoProps> = ({ date, fallback = '', live }) => {
-    const getValue = () => {
-        if (!date) return fallback;
+const TimeAgo: FC<TimeAgoProps> = ({
+    date,
+    fallback = '',
+    refresh = true,
```

Comment:
same point as before: it's weird to have an optional boolean prop default to `true`. We don't do that anywhere else as far as I'm aware. Would it make sense to use "negative" here instead, e.g. "dontRefresh", so that `undefined` and `false` have the same meaning?

Reply:
We have it in
- `frontend/src/component/feature/StrategyTypes/FlexibleStrategy/FlexibleStrategy.tsx`
- `frontend/src/component/project/ProjectList/ProjectGroup.tsx`
- `frontend/src/component/common/FeatureStatusChip/FeatureStatusChip.tsx`
- `frontend/src/component/changeRequest/ChangeRequest/Changes/Change/StrategyChange.tsx`